src/program.jl

```diff
@@ -34,10 +32,9 @@ Base.pointer(p::Program) = p.id
 
 Base.getindex(p::Program, pinfo::Symbol) = info(p, pinfo)
 
-function Program(ctx::Context; source=nothing, binaries=nothing)
-    local program_id::cl_program
-    if source !== nothing && binaries !== nothing
-        throw(ArgumentError("Program be source or binary"))
+function Program(ctx::Context; source=nothing, binaries=nothing, il=nothing)
+    if source !== nothing && binaries !== nothing && il !== nothing
```

Comment:
Isn’t the logic here to catch more than two valid inputs? I don’t think this is doing it anymore.

Reply:
Ah yeah, that's broken now. Fix in https://github.com/JuliaGPU/OpenCL.jl/pull/213